### DISCUSSION TITLE
Unable to change Codespaces Ownership

### DISCUSSION THREAD

**Discussion by steven-tomlinson** [community/community#120157] [Codespaces]
### Select Topic Area

Bug

### Body

When attempting to change the setting for Codespaces Ownership, I am unable to Submit the changes from the pop-up warning me about the change which I am attempting to perform.

When I click the Submit button, it just remains high-lighted and the pop-up is not dismissed.
I'm using the current version of Microsoft Edge on the current version of Microsoft Windows 11.

![image](https://github.com/community/community/assets/8976999/d4b42523-e85f-46c5-9fba-dc4975df1c76)

**Comment by riowaller** [comment]
Hello, I've also gotten this error.  Did you figure it out? Thanks for your time!
